File: sort_associative_array_by_value_descend/sort_associative_array_by_key_descend.js
let example_array = new Map( [ ["January", 1], ["December", 12], ["March", 3] ].sort((a, b) => {a = a[1]; b = b[1]; return b - a;}) );
for (let [key, value] of example_array.entries()) {
  console.log(key + ' => ' + value);
}

File: examples/array/sort_associative_array_by_value_descend/sort_associative_array_by_key_descend.rb
example_array = {"January" => 1, "December" => 12, "March" => 3 }
sorted_array = Hash[ example_array.sort_by{ |_, v| -v } ]
sorted_array.each { |tuple| print([tuple[0].to_s, tuple[1].to_s].join(" => ")+"\n") }
